commit 6ec4e729e2426dac37e9e27932d4d9e579d53ba4
Author: Vishesh Handa <handa.vish@gmail.com>
Date:   Fri May 25 19:57:26 2012 +0530

    Do not compile and install the rcgen
    
    Conflicts with kdelibs.

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5c7c8ab..3bd49c2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -57,7 +57,9 @@ include_directories(
 )
 
 add_subdirectory(libnepomukcore)
-add_subdirectory(rcgen)
+# Do not install rcgen, it's installed by kdelibs
+# If it needs to be installed then we need to change the name
+#add_subdirectory(rcgen)
 add_subdirectory(ontologies)
 add_subdirectory(common)
 add_subdirectory(server)
